### DISCUSSION TITLE
creating-studies.html

### DISCUSSION THREAD

**Discussion by alpivonka** [smart-on-fhir/cumulus#15] [General]
As I am reading through your documentation to become more formulae with the setup and configuration. I ran into an issue not being able to find a resource you call "Cumulus library template repo" as it links to a non-existing github location. 

Many thanks

**Comment by mikix** [comment]
Hello! Good eye and thanks for your report - [that repo](https://github.com/smart-on-fhir/cumulus-library-template) exists, but it's marked as private.

@dogversioning do you know if there's a reason we couldn't open that up?

**Reply by dogversioning** [reply]
I am 95% sure it's ok but we'll need to get an official sign off before we open it up, which may take a day or two due to vacations

In the interim - the code content of that repo is largely the same as the [template study directory](https://github.com/smart-on-fhir/cumulus-library/tree/main/cumulus_library/studies/template) in the currently public repo.

**Comment by dogversioning** [comment]
Ok, that was a bit faster than I'd hoped! @alpivonka that repo should now be public. Holler at if you run into any other issues.
